### PR TITLE
tracer: add offset time from when a span starts to logs

### PR DESCRIPTION
**Pull request by patrickkenney9801**
Adds more metadata to logs so devs can quickly see how long a span took

### CODE REVIEW

File: libsupport/src/JSONTracer.cpp [hunk 1]
```diff
@@ -114,15 +114,15 @@ std::string
...
+    const std::string& host_data, uint64_t duration) {
   fmt::memory_buffer buf;
   uint32_t host_id = katana::ProgressTracer::Get().GetHostID();
   static auto begin = katana::Now();
   auto msec_since_begin = katana::UsSince(begin) / 1000;
 
   fmt::format_to(
-      std::back_inserter(buf), R"({{"host":{},"offset_ms":{})", host_id,
-      msec_since_begin);
+      std::back_inserter(buf), R"({{"host":{},"offset_ms":{},"span_ms":{})",
```
Comment (ddn0) [inline libsupport/src/JSONTracer.cpp:124]: I get how this makes it easier to read the raw traces, but why wouldn't we just make the tools that parse traces easier to use?

If people find using Jaeger inconvenient to view these traces, I'd rather just write our own TUI frontend to raw trace data than conflate a trace (data) with how it is viewed or analyzed.

Comment (patrickkenney9801) [reply libsupport/src/JSONTracer.cpp:124]: @l-hoang is the primary asker for this feature

Comment (patrickkenney9801) [reply libsupport/src/JSONTracer.cpp:124]: And the offset_ms fields are also just cruft to make the logs easier to use and aren't used to assemble traces

**Comment by patrickkenney9801** [comment]
Closed waiting further review on adding features to tracing

**Pull request closed without merging**